codechain patchfile version 2
treehash 2072327b4f29f250f248031845bd1fae52b73f7435a2dabad4de1c545bcc7f1b
- f cef835dae8ba0a47b6fd196099a6865d9142ab00e1ce9e22e4f12bc9bd2f9cc9 .testnet.secpkg
+ f 92e9b3be71984557998de94c5981d9766e06e155e87400981cd52b5dadcfe451 .testnet.secpkg
dmppatch 5
@@ -25,86 +25,86 @@
 t%22,%0A
-  %22Head%22: %22bfe4ef8bcca94c6a7d22a266b62ccca437a9dbd1a19d7670051d86448631a733%22,%0A
+  %22Head%22: %22039da38b8ff489908f9d796519a1a8608191fdfb47d791b45ed7c45beff82e91%22,%0A
   %22D
- f 59e463dc0aa499cc9f4b7621f8e6eab19424133fc117b394646035a3d4593aee updater/updatemanager.go
+ f 843795eb7df067d43d82b2c5cfe0e7bff9cc33eff6280ee8879a8efc80a5b015 updater/updatemanager.go
dmppatch 13
@@ -905,16 +905,50 @@
    bool%0A
+%09updateTime             time.Time%0A
 %7D%0A%0Afunc 
@@ -3759,106 +3759,135 @@
 )))%0A
-%09log.Warnf(%22Exiting for update at %25s%22,%0A%09%09time.Now().UTC().Add(timeToUpdate).Format(time.RFC3339))%0A
+%09um.updateTime = time.Now().UTC().Add(timeToUpdate)%0A%09log.Warnf(%22Exiting for update at %25s%22, um.updateTime.Format(time.RFC3339))%0A
 %0A%09go
@@ -4181,16 +4181,106 @@
 )%0A%09%09%09%09%7D%0A
+%09%09%09%7D else %7B%0A%09%09%09%09log.Warnf(%22Exiting for update at %25s%22, um.updateTime.Format(time.RFC3339))%0A
 %09%09%09%7D%0A%09%09%7D
treehash de7398f7ea88d34ff4cf7c20a709ea6ef1c93af5926a7a0f81fae5fba4067d72
